ipa-core/src/secret_sharing/mod.rs

```diff
@@ -62,7 +62,7 @@ pub trait Block: Sized + Copy + Debug {
     type Size: ArrayLength;
 }
 
-pub trait Sendable: Send + Debug + Serializable + 'static {}
+pub trait Sendable: Send + Sync + Debug + Serializable + 'static {}
```

Comment:
curious what drives this to be `Sync` as well?

Reply:
It is a consequence of the change to `mask_and_shuffle` to accept borrowed input. For the input `&T` to `mask_and_shuffle` to be `Send`, `T` needs to be `Sync`.